ff-campaign/index.html

```diff
@@ -7,7 +7,8 @@
     <link rel="stylesheet" href="styles.css">
 
     <script>
-        document.addEventListener("ffReady", function () {
+        document.addEventListener("ffReady", function (event) {
+            const factfinder = event.factfinder;
```

Comment:
Shall we use `event.resultDispatcher` here like in the other demos?

Reply:
my mistake, It's fixed already